apps/livreGUI/tfEditor/ColorMapWidget.h

```diff
@@ -21,7 +21,7 @@
 #ifndef _ColorMapWidget_h_
 #define _ColorMapWidget_h_
 
-#include <livreGUI/tfEditor/TFWidget.h>
+#include <livreGUI/tfEditor/TFWidget.h> // base class
```

Comment:
Why do we need this comment ? Maybe change class name if it is creating confusion.